book/chapters/appendices/solutions.qmd

```diff
@@ -363,8 +361,7 @@ instance = OptimInstanceMultiCrit$new(
 Using the surrogate, acquisition function and acquisition function optimizer that are provided, we can proceed to optimize the instance via ParEGO:
 
 ```{r}
-surrogate = srlrn(lrn("regr.ranger", num.trees = 10L, keep.inbag = TRUE,
-  se.method = "jack"))
+surrogate = srlrn(default_rf())
```

Comment:
I don't think adding these functions here makes sense as you only have the small note below.  Either we need to expand upon the note more or go back to what we had before

Reply:
Only included this to show where it goes but BB told me that exercises and solutions will be rewritten by him and Martin.